Source/WTF/wtf/FunctionDispatcher.h

```diff
@@ -52,7 +52,8 @@ class WTF_CAPABILITY("is current") WTF_EXPORT_PRIVATE SerialFunctionDispatcher :
 // A RefCountedSerialFunctionDispatcher guarantees that a dispatched function will always be run.
 class RefCountedSerialFunctionDispatcher : public SerialFunctionDispatcher {
 public:
-    DECLARE_VIRTUAL_REFCOUNTED;
+    virtual void ref() const = 0;
+    virtual void deref() const = 0;
```

Comment:
I skipped the base class here for now because I got a cryptic linker error about

> no declaration found for exported symbol 'non-virtual thunk...'

that I couldn't figure out.